### PR TITLE
feat: Store each INDIVIDUAL email as "PDF" instead of an entire thread

### PR DESCRIPTION
**Pull request by piraveen**

### CODE REVIEW

File: Config.gs [hunk 1]
```diff
@@ -58,13 +58,20 @@ function getGmail2GDriveConfig() {
...
+        // Store threads marked with label "PDF" in the folder "PDF Emails" as PDF document.
         "filter": "label:PDF",
         "saveThreadPDF": true,
         "folder": "PDF Emails"
       },
       {
-        // Store threads marked with label "PDF" in the folder "PDF Emails" als PDF document.
+        // Store each INDIVIDUAL email as "PDF" instead of an entire thread, in the folder "PDF Emails"
+        "filter": "from:no_reply@email-invoice.example.com",
+        "saveMessageAsPDF": true,
```
Comment (ahochsteger) [inline Config.gs:69]: @piraveen for consistency I'd name it `saveMessagePDF` to match `saveThreadPDF`.

File: Config.gs [hunk 2]
```diff
@@ -58,13 +58,20 @@ function getGmail2GDriveConfig() {
...
+        "skipHeader": true, // Skip Email Header
```
Comment (ahochsteger) [inline Config.gs:70]: @piraveen since this is meant for PDF export only, I'd name it `skipPDFHeader`.

File: Config.gs (hunk 1 above)
Comment (piraveen) [reply Config.gs]: @ahochsteger Thx for the feedback, changes updated in commit: [4db6634](https://github.com/ahochsteger/gmail2gdrive/pull/73/commits/4db6634c69d87888ae494b327e958c9deab545d2)

File: Config.gs (hunk 2 above)
Comment (piraveen) [reply Config.gs]: @ahochsteger Thx for the feedback, changes updated in commit: [4db6634](https://github.com/ahochsteger/gmail2gdrive/pull/73/commits/4db6634c69d87888ae494b327e958c9deab545d2)

**Comment by piraveen** [comment]
@ahochsteger also added the missing function `processMessageToHtml` in commit: [4db6634](https://github.com/ahochsteger/gmail2gdrive/pull/73/commits/4db6634c69d87888ae494b327e958c9deab545d2)

**Review by ahochsteger — APPROVED** [review]
@piraveen looks good to me, thanks for your contribution!

**Pull request merged**